components/multiselect/multiselect_list.tsx

```diff
@@ -143,46 +147,94 @@ export default class MultiSelectList extends React.Component<Props, State> {
         );
     }
 
-    public render() {
-        const options = this.props.options;
-        let renderOutput;
+    private renderOptions(options: Value[]) {
+        let renderer: Props['optionRenderer'];
```

Comment:
i am 0/5 but this can be simplified as `cont renderer = this.props.optionRenderer || this.defaultOptionRenderer`?.